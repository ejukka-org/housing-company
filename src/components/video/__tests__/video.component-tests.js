import React from 'react';
import Video from '../video-component';
import renderer from 'react-test-renderer';

test('Renders Video component', () => {
    const component = renderer.create(
        <Video />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
