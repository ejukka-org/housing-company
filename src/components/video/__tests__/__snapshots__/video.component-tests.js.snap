// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Video component 1`] = `
<div
  className="container"
>
  <video
    controls={true}
    height="240"
    hidden={false}
    poster="talo1.png"
    width="320"
  >
    <source
      src="https://media.w3.org/2010/05/sintel/trailer_hd.mp4"
      type="video/mp4"
    />
    Your browser does not support the video tag.
  </video>
</div>
`;
